.gitignore

```diff
@@ -17,4 +17,9 @@ __pycache__/
 *$py.class
 /.atom-build.json
 .vscode/
-.vs/
\ No newline at end of file
+.vs/
+config/example/admins.txt
+config/example/admins.txt
+config/example/alienwhitelist.txt
+config/example/admins.txt
+config/example/config.txt
```

Comment:
ditto

Reply:
Everything in gitignore are for my test server and the test server alone, I'll remove them in the next push.